string                   map_id
geographic_msgs/GeoPoint geographic_point
geometry_msgs/Quaternion orientation_3d